account_payment_sale/models/sale_order.py

```diff
@@ -31,11 +32,11 @@ def _get_payment_mode_vals(self, vals):
                 and self.payment_mode_id.payment_method_id.code == "manual"
             ):
                 vals[
-                    "invoice_partner_bank_id"
+                    "partner_bank_id"
                 ] = self.payment_mode_id.fixed_journal_id.bank_account_id.id
-        return vals
 
     def _prepare_invoice(self):
         """Copy bank partner from sale order to invoice"""
         vals = super()._prepare_invoice()
-        return self._get_payment_mode_vals(vals)
+        self._get_payment_mode_vals(vals)
```

Comment:
Strange previous structure, but as you are changing this, please merge both methods into one and directly change the `vals` dictionary. The inheritance won't suffer by this change.

Reply:
I don't understand your remark. What would you like exactly ?